rubicon/objc/objc.py

```diff
@@ -948,8 +948,7 @@ def setter(_self, new):
         def _objc_getter(objc_self, objc_cmd, *args):
             from .core_foundation import at
             py_self = ObjCInstance(objc_self)
-            args = convert_method_arguments(getter_encoding, args)
-            result = getter(py_self, *args)
+            result = getter(py_self, *[ObjCInstance(a) for a in args])
```

Comment:
I'm a bit confused why the `*args` are needed for the getter and setter. A getter always takes no arguments, and a setter always takes exactly one argument, right?

Reply:
Yeah - I couldn't think of any reason that `*args` would be used in a getter; I preserved it because it was effectively a no op. But I suppose not passing the argument at all would be even more of a no-op :-)

The setter can also be cleaned up in the same vein, since it should always have exactly 1 argument.